The description of the command is too long

Here are the server logs indicating that the description of `/skill` is too long:
```
2023-09-05T02:55:35.867566727+00:00 [LOG]   Registered /skill.
2023-09-05T02:55:36.135524441+00:00 [ERROR] Invalid Form Body (7.description: Must be 100 or fewer in length.)
2023-09-05T02:55:36.135524441+00:00 [ERROR]   because: Invalid Form Body (7.description: Must be 100 or fewer in length.)
```

--- a/src/commands/skill.rs
+++ b/src/commands/skill.rs
@@ -187,7 +187,7 @@ impl SkillCommand {
 impl BotCommand for SkillCommand {
     fn register(&self, command: &mut CreateApplicationCommand) {
         command
-            .description("Does a skill roll following the Call of Cthulhu 6th Edition. `/sk6` is the same. | 第6版のルールに基づいて技能ロールを行います. `/sk6`と同じ動きをします.")
+            .description("Does a skill roll. `/sk6` (The CoC 6th Edition) is the same. | `/sk6`と同様に, 第6版のルールに基づいて技能ロールを行います.")
             .create_option(|option| {
                 option
                     .name("chance")

